client/my-sites/marketing/main.jsx

```diff
@@ -136,7 +136,7 @@ export default connect( state => {
 
 	return {
 		showButtons: siteId && canManageOptions && ( ! isJetpack || hasSharedaddy ),
-		showConnections: ! siteId || ! isJetpack || isJetpackModuleActive( state, siteId, 'publicize' ),
+		showConnections: ! siteId || ! isJetpack,
```

Comment:
I think we only need to check for the presence of the `siteId` here, as all sites now need to see this section, regardless of how publicize is set in Jetpack wp-admin.

```suggestion
		showConnections: siteId,
```